boards/xtensa/esp32/doc/esp32.rst

```diff
@@ -62,6 +62,46 @@ The toolchain is available for Linux, Windows, and Mac hosts and
 instructions to obtain and set them up are available in the ESP-IDF
 repository, using the toolchain and SDK links above.
 
+Docker container based builds
+=============================
+
+You can skip locally installing and configuring the Zephyr build system by
+using pre-made the ``svendowideit/zephyr-esp32`` Docker image.
```

Comment:
I suspect the Zephyr development team may want to provide a validated docker image rather than including personal user images in the official Zephyr documentation (but I'll leave it to them to decide).  From a tech writer point of view though, I'd recommend changing this to:

    using a pre-made Docker image ``svendowideit/zephyr-esp32``

Reply:
yup, I would expect that _if_ this PR is desired, then the image should be moved to a zephyrproject user - for now, I'm using my own, so i can try it out ;)